inc/item_operatingsystem.class.php

```diff
@@ -41,6 +41,7 @@ class Item_OperatingSystem extends CommonDBRelation {
    static public $itemtype_2 = 'itemtype';
    static public $items_id_2 = 'items_id';
    static public $checkItem_1_Rights = self::DONT_CHECK_ITEM_RIGHTS;
+   static public $checkItem_2_Rights = self::DONT_CHECK_ITEM_RIGHTS;
```

Comment:
I am not really confident with this.

Reply:
I've reverted that change; I've made a small change on inventory regarding entity stored in session; and that fixed it.